src/data/pi.lean

```diff
@@ -18,6 +19,23 @@ variables (x y : Π i, f i) (i : I)
 
 namespace pi
 
+/-- `1`, `0`, `+`, `*`, `-`, `⁻¹`, and `/` are defined pointwise. -/
```

Comment:
```suggestion
/-! `1`, `0`, `+`, `*`, `-`, `⁻¹`, and `/` are defined pointwise. -/
```

But let's do this just before triggering bors, since there's no need to rerun the build with that change.